terraform/github/locals.tf

```diff
@@ -15,7 +15,8 @@ locals {
     "ewastempel",
     "jakemulley",
     "SteveMarshall",
-    "davidkelliott"
+    "davidkelliott",
+    "connormaglynn"
```

Comment:
☝️ I'm an Organisation owner, so I need to be on the maintainers' list 🔧

Reply:
ok